tests/test_wog.py

```diff
@@ -114,6 +114,22 @@ def test_chogall_cannot_pay_health():
 	assert not fireball.is_playable()
 
 
+def test_demented_frostcaller():
+	game = prepare_game()
+	game.player1.give("OG_085").play()
+	game.end_turn()
+
+	game.player2.give(WISP).play()
+	game.end_turn()
+
+	enemies = game.player2.characters
+	for x in range(1, 3):
+		game.player1.give(THE_COIN).play()
+		assert len([en for en in enemies if en.frozen]) == x
```

Comment:
You can do `assert len(game.player2.characters.filter(frozen=True)) == i` instead.